testCourse/questions/truthTableDemo/server.py

```diff
@@ -0,0 +1,10 @@
+def generate(data):
+    variables = [{"name": "A"}, {"name": "B"}, {"name": "C"}, {"name": "D"}]
+
+    rows = []
+
+    rows.append({"C": 0, "D": 1})
+
+    data["params"]["variables"] = variables
+    data["params"]["rows"] = rows
+    data["params"]["expression"] = "A or B"
```

Comment:
Is there a way to use the element without needing to use a `server.py`? It would be nice to have a way to do that, and an example of it.

Reply:
ah, ok, I see in the example question, one of the examples uses the `server.py` and one of them doesn't. That's great.